sdk/eventgrid/azure-messaging-eventgrid-perf/src/main/java/com/azure/messaging/eventgrid/perf/SendCloudEventsTest.java

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.messaging.eventgrid.perf;
+
+import com.azure.messaging.eventgrid.CloudEvent;
+import com.azure.perf.test.core.PerfStressOptions;
+import reactor.core.publisher.Mono;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class SendCloudEventsTest extends ServiceTest<PerfStressOptions> {
+    public SendCloudEventsTest(PerfStressOptions options) {
+        super(options);
+    }
+
+    @Override
+    public void run() {
+        cloudEventPublisherClient.sendCloudEvents(createEvents());
```

Comment:
The create Events method needs to be moved to setupAsync method or the constructor, store the events as a local Array List.